modal/running_app.py

```diff
@@ -0,0 +1,15 @@
+# Copyright Modal Labs 2024
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+
+from google.protobuf.message import Message
+
+
+@dataclass
+class RunningApp:
```

Comment:
Full circle! Didn't we use to have something called `RunningApp`, that was the context var of the stub.run context manager (which was then `app.run()` iirc) ? :D

Reply:
I think that was basically the `App` class, which I split up at some point, then eventually merged back (after deleting like 99% of the code however)